.github/actions/restore_dbt_cache/action.yaml

```diff
@@ -0,0 +1,76 @@
+name: Restore dbt cache
+description: Attempts to restore dbt cache from S3 storage.
+inputs:
+  path:
+    description: The local path to restore the cache to in case of a hit.
+    required: true
+  key:
+    description: The cache key to query for.
+    required: true
+  restore-key:
```

Comment:
The [GitHub Action](https://github.com/actions/cache) that this action is based on accepts `restore-keys` as a newline-delimited list, but for our purposes I only ever expect we'll need to use one fallback (for the master cache), so we gain some simplicity by switching to accepting a string value.